### DISCUSSION TITLE
New Images

### DISCUSSION THREAD

**Discussion by alfista-sk** [mattieserver/netbox-topology-views#537] [Ideas]
### NetBox version

v4.x

### Topology Views version

v4.x

### Feature type

New functionality

### Proposed functionality

Is possible to add there more device images, like for virtualisation (VM, Host, ... - as inspiration you can use pictograms from VMware), maby also in more colors (red, blue, green, yellow) and kubernetes?

### Use case

We have mixed infrastructure and almost all is in virtualisation and kubernetes.

### External dependencies

As ilustration you can use pictograms from VMware for virtual infrastructure and for kubernetes are there also many nice samples.

**Comment by mattieserver** [comment]
That's kinda the purpose of the ' preferences - images' feature. You can upload any icon you want in your own netbox based on the device role.

Unless you actually want to create the images and contribute them to this project I don't see what we can do for you.

**Comment by alfista-sk** [comment]
OK thanks understand.
